pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```diff
@@ -31,8 +34,25 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ];
 
   buildInputs = [
-    indilib libnova curl cfitsio libusb1 zlib boost gsl gpsd
-    libjpeg libgphoto2 libraw libftdi1 libdc1394 ffmpeg fftw
+    indilib
+    libnova
+    curl
+    cfitsio
+    libusb1
+    systemd
+    zlib
+    boost
+    pkg-config
```

Comment:
pkg-config goes into nativeBuildInputs

Reply:
can you please provide more information, what is NativeBuildInputs?

Reply:
https://nixos.org/manual/nixpkgs/unstable/#variables-specifying-dependencies